pkg/ddc/jindo/transform.go

```diff
@@ -310,22 +310,23 @@ func (e *JindoEngine) transformFuse(runtime *datav1alpha1.JindoRuntime, value *J
 
 func (e *JindoEngine) transformFuseNodeSelector(runtime *datav1alpha1.JindoRuntime, value *Jindo) (err error) {
 	value.Fuse.NodeSelector = map[string]string{}
-	if runtime.Spec.Fuse.Global {
-		value.Fuse.Global = true
-		if len(runtime.Spec.Fuse.NodeSelector) > 0 {
-			value.Fuse.NodeSelector = runtime.Spec.Fuse.NodeSelector
-		}
-		value.Fuse.NodeSelector[common.FLUID_FUSE_BALLOON_KEY] = common.FLUID_FUSE_BALLOON_VALUE
-		e.Log.Info("Enable Fuse's global mode")
-	} else {
-		if runtime.Spec.Fuse.NodeSelector == nil {
-			labelName := e.getCommonLabelname()
-			value.Fuse.NodeSelector[labelName] = "true"
-			e.Log.Info("Disable Fuse's global mode")
-		} else {
-			value.Fuse.NodeSelector = runtime.Spec.Fuse.NodeSelector
-		}
-	}
+	value.Fuse.NodeSelector[e.getFuseLabelname()] = "true"
```

Comment:
I think we should also support the user defined node label?

Reply:
Done. Thx